docs/examples/descriptions/customized-style.vue

```diff
@@ -25,10 +25,10 @@
   </el-descriptions>
 </template>
 <style scoped>
-.my-label {
-  background: var(--el-color-success-light-9);
+:deep(.my-label ){
```

Comment:
Maybe you should remove the spaces between letters and brackets.

Reply:
Yeah, I didn't enable the eslint plugin while I was fixing the example, which is causing the code style problem. I've force-pushed a new commit, please take a look.